packer/setup_root.sh

```diff
@@ -66,6 +61,12 @@ spec:
   valuesContent: |-
     controller:
       watchIngressWithoutClass: true
+      image:
+        digest: ""
```

Comment:
Unfortunately, offline loading of the image changes the digest which were causing downloading images still